lsp-haxe.el

```diff
@@ -61,6 +61,11 @@
   :risky t
   :type '(repeat string))
 
+;; The build spec for the project.
+(defcustom lsp-haxe-hxml "build.hxml"
+  "The compile file for the haxe project."
+  :type 'file)
```

Comment:
nit: can you add `:package-version '(lsp-mode . "6.4")`

Reply:
Thanks for the feedback! 

I've included the `package-version` and addressed the new notifications that the haxe language server was sending.